public/components/common/form/index.tsx

```diff
@@ -8,6 +8,22 @@ import { InputFormFilePicker } from './input_filepicker';
 import { InputFormTextArea } from './input_text_area';
 import { EuiFlexGroup, EuiFlexItem, EuiFormRow } from '@elastic/eui';
 
+interface InputFormProps {
+  type: string;
```

Comment:
**nitpick**: This could be enhanced by defining the expected values instead of a generic string.